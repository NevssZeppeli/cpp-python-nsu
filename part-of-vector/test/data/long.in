18
11
99
-13
-20
37
63
10
82
-11
21
-6
51
68
29
93
7
-4
29
58
10
86
68
39
-3
-17
-9
-9
-11
32
61
9
85
61
76
42
33
31
61
22
71
55
88
36
24
99
62
7
-7
58
40
-20
-13
2
68
28
29
-5
70
93
-11
-9
87
65
53
85
37
6
-3
37
-6
92
-6
11
51
22
-7
-1
43
74
67
-20
37
74
59
92
28
-13
30
93
58
91
-9
47
56
9
96
91
7
13
47
69
-10
2
50
61
47
44
93
53
-6
44
58
-17
11
65
82
22
65
-6
8
-16
64
-4
60
93
8
68
64
97
20
2
-17
9
95
66
40
38
70
67
61
38
70
86
41
64
46
-12
90
51
26
74
71
17
22
76
34
31
44
67
98
56
33
59
17
70
-8
-7
81
-7
3
13
61
41
20
-10
-12
68
50
69
79
23
16
99
54
44
44
52
2
12
46
-16
62
11
83
-13
25
80
18
28
7
38
91
70
-1
43
37
53
49
-1
79
95
15
16
34
31
65
7
82
-3
-12
42
99
94
82
82
-17
2
-9
44
50
33
52
59
-13
18
19
-1
78
55
42
81
78
75
-11
100
4
75
100
79
76
87
90
45
21
96
61
73
50
26
35
24
81
39
-2
90
-2
47
52
53
63
69
24
2
60
14
59
74
67
66
-6
80
2
73
47
84
22
8
60
76
90
9
50
98
-11
96
5
71
31
45
-15
56
-7
5
31
92
21
38
23
73
49
30
-17
48
74
45
26
43
86
78
71
9
90
49
11
31
59
41
25
-20
42
-19
54
68
-20
63
26
19
65
86
15
35
49
96
54
-2
19
29
39
23
89
90
63
75
7
1
18
48
20
79
51
22
24
38
-19
16
51
10
60
50
43
5
59
77
36
70
72
40
51
14
69
32
95
-2
47
85
69
-3
-1
100
-17
89
-10
99
90
58
30
86
24
71
92
11
71
46
80
31
99
23
37
39
46
27
29
79
46
100
84
-8
-16
-6
-8
55
39
3
80
17
51
28
-6
7
-8
-6
29
44
69
58
87
89
35
97
23
46
71
5
-12
-8
-10
73
16
76
96
1
53
5
34
88
15
45
100
-20
26
41
37
59
8
33
25
7
34
11
17
-16
53
-2
78
77
18
-1
39
74
40
86
46
22
65
85
26
-17
-5
57
73
25
-15
47
80
32
59
32
50
-18
40
59
28
79
97
1
42
1
19
31
47
25
38
41
20
50
20
32
87
58
66
74
89
49
74
-12
74
99
63
40
62
-2
64
1
8
78
-13
53
4
44
83
39
3
58
41
34
37
94
73
50
73
12
-12
24
46
-20
-6
46
91
87
91
69
15
43
23
78
55
28
85
18
17
95
30
6
37
98
82
88
59
25
83
99
69
50
70
69
-1
84
28
49
-3
82
57
38
59
26
50
43
40
86
8
-4
-2
83
89
90
51
68
100
60
50
77
71
52
61
100
75
57
63
20
0
-16
54
87
-2
78
7
-6
27
27
85
36
-2
92
74
49
77
26
84
-14
-6
6
60
75
12
29
25
11
-8
-13
-10
-1
49
-4
11
24
14
77
30
-14
-12
87
66
58
88
77
52
-7
-17
38
33
7
-18
26
58
44
13
83
-9
57
61
18
75
-2
72
53
15
-18
-14
67
89
57
86
82
26
21
-17
87
67
68
95
32
11
4
74
93
99
55
44
44
95
86
77
99
45
29
40
17
70
62
8
84
75
81
16
-8
-8
-9
24
36
84
36
62
42
65
70
84
11
61
-17
62
4
25
32
85
20
-19
6
68
14
38
77
-1
-3
10
7
74
61
70
23
82
63
91
21
-5
89
91
8
57
-18
1
70
92
2
38
89
21
-14
65
100
93
27
58
23
14
-13
19
69
71
56
5
21
73
31
17
51
-7
80
83
90
-11
4
75
-3
2
18
92
3
-10
23
15
92
-4
92
-16
59
100
54
41
11
46
82
-5
2
63
17
100
98
13
58
89
80
60
-14
-12
21
66
3
19
63
32
3
-13
-19
42
34
7
30
13
51
67
59
49
46
13
49
98
83
91
-3
16
98
63
70
-16
9
76
28
-13
97
-11
22
43
29
98
86
24
24
-6
89
6
78
65
3
26
69
49
70
19
-8
78
26
38
100
90
18
4
-12
19
-8
94
49
84
-7
51
78
28
-8
22
21
-19
86
-17
74
25
85
46
98
29
37
70
76
42
76
-4
85
57
81
-15
48
81
-16
55
2
33
-9
86
27
35
94
54
91
48
86
99
86
52
85
-2
36
-3
78
87
71
82
41
39
80
-16
18
12
60
-5
15
-17
2
66
74
58
42
61
16
61
9
72
62
87
54
46
48
71
63
57
36
98
37
72
73
27
32
43
26
-10
10
18
73
41
-7
0
19
