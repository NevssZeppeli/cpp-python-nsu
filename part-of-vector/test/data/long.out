99 11 18